openassessment/assessment/models/ai.py

```diff
@@ -25,18 +25,12 @@
 # The classifier data will generally be larger than memcached's default max size
 CLASSIFIERS_CACHE_IN_MEM = getattr(
     settings, 'ORA2_CLASSIFIERS_CACHE_IN_MEM',
-    get_cache(
-        'django.core.cache.backends.locmem.LocMemCache',
-        LOCATION='openassessment.ai.classifiers_dict'
-    )
+    caches['cache_in_mem']
 )
 
 CLASSIFIERS_CACHE_IN_FILE = getattr(
     settings, 'ORA2_CLASSIFIERS_CACHE_IN_FILE',
-    get_cache(
-        'django.core.cache.backends.filebased.FileBasedCache',
-        LOCATION='/tmp/ora2_classifier_cache'
-    )
+    caches['cache_in_file']
```

Comment:
@muhammad-ammar If `_create_cache` allows us to retain the existing behaviour, lets use it.